src/foundation/reboot.css

```diff
@@ -0,0 +1,585 @@
+  background-color: var(--body-bg); // 2
+  -webkit-text-size-adjust: 100%; // 3
+  -webkit-tap-highlight-color: rgba(var(--black), 0); // 4
+}
+
+// Future-proof rule: in browsers that support :focus-visible, suppress the
+// focus outline on elements that programmatically receive focus but
+// wouldn't normally show a visible focus outline. In general, this would
+// mean that the outline is only applied if the interaction that led to the
+// element receiving programmatic focus was a keyboard interaction, or the
+// browser has somehow determined that the user is primarily a keyboard
+// user and/or wants focus outlines to always be presented.
+//
+// See https://developer.mozilla.org/en-US/docs/Web/CSS/:focus-visible
+// and https://developer.paciellogroup.com/blog/2018/03/focus-visible-and-backwards-compatibility/
+
+[tabindex='-1']:focus:not(:focus-visible) {
+  outline: 0 !important;
+}
+
+// Content grouping
+//
+// 1. Reset Firefox's gray color
+// 2. Set correct height and prevent the `size` attribute to make the `hr`
+//    look like an input field
+//    See https://www.w3schools.com/tags/tryit.asp?filename=tryhtml_hr_size
+
+hr {
+  margin: var(--hr-margin-y) 0;
+  color: var(--hr-color); // 1
+  background-color: currentColor;
+  border: 0;
+  opacity: var(--hr-opacity);
+}
+
+hr:not([size]) {
+  height: var(--hr-height); // 2
+}
+
+// Typography
+//
+// 1. Remove top margins from headings
+//    By default, `<h1>`-`<h6>` all receive top and bottom margins.
+//    We nuke the top margin for easier control within type scales
+//    as it avoids margin collapsing.
+
+%heading {
+  margin-top: 0; // 1
+  margin-bottom: var(--headings-margin-bottom);
+  font-family: var(--headings-font-family);
+  font-style: var(--headings-font-style);
+  font-weight: var(--headings-font-weight);
+  line-height: var(--headings-line-height);
+  color: var(--headings-color);
+}
+
+h1 {
+  @extend %heading;
+  font-size: var(--h1-font-size);
+}
+
+h2 {
+  @extend %heading;
+  font-size: var(--h2-font-size);
+}
+
+h3 {
+  @extend %heading;
+  font-size: var(--h3-font-size);
+}
+
+h4 {
+  @extend %heading;
+  font-size: var(--h4-font-size);
+}
+
+h5 {
+  @extend %heading;
+  font-size: var(--h5-font-size);
+}
+
+h6 {
+  @extend %heading;
+  font-size: var(--h6-font-size);
+}
+
+// Reset margins on paragraphs
+//
+// Similarly, the top margin on `<p>`s get reset. However, we also reset the
+// bottom margin to use `rem` units instead of `em`.
+
+p {
+  margin-top: 0;
+  margin-bottom: var(--paragraph-margin-bottom);
+}
+
+// Abbreviations
+//
+// 1. Duplicate behavior to the data-* attribute for our tooltip plugin
+// 2. Add the correct text decoration in Chrome, Edge, IE, Opera, and Safari.
+// 3. Add explicit cursor to indicate changed behavior.
+// 4. Prevent the text-decoration to be skipped.
+
+abbr[title],
+abbr[data-original-title] {
+  // 1
+  text-decoration: underline; // 2
+  text-decoration: underline dotted; // 2
```

Comment:
If you see more than one CSS property it's always because of browser support. All browsers apply the first one. However, second one will be applied only if browser understands that.